src/routes/townhalls/rating/index.ts

```diff
@@ -0,0 +1,43 @@
+import { Router } from 'express';
+import { ObjectId } from 'mongodb';
+import Joi from 'joi';
+import type { RatingForm, Rating, User } from 'prytaneum-typings';
+import { ratingValidationObject } from 'modules/townhall/validators';
+import API from 'modules/rating';
+import JWT from 'lib/jwt';
+
+import { makeJoiMiddleware, makeEndpoint, getCookies } from 'middlewares';
+import { TownhallParams, RatingParams } from '../types';
+
+const router = Router();
+
+router.put<TownhallParams, void, RatingForm, RatingParams>(
```

Comment:
Since we're always adding a new rating, then it should be a POST, not PUT.  PUT is idempotent meaning the request should result in the same state no matter how many calls.  This is not the case here. Calling this endpoint will result in additional ratings being added every single time.  

An example of an idempotent endpoint is submitting an update to a form.  If the form fields are sent every time and it's just a mongodb $set, then that is idempotent.  Calling it once versus 100 times results in the same thing.